create_scaling_table_per_exp.py

```diff
@@ -153,13 +153,21 @@ def grep(string, filename):
         list_line = []
         list_iline = []
         lo_success = False
-
-        for iline, line in enumerate(open(filename)):
-            if string in line:
-                list_line.append(line)
-                list_iline.append(iline)
-                lo_success = True
-
+        file = open(filename, 'r')
+        count = 0
+        while True:
+            try:  # Some lines are read in as binary with the pgi compilation
+                line = file.readline()
+                count = +1
```

Comment:
Script is working fine now, but this should be `count += 1`

Reply:
Thanks, I adapted it!